singleFrameProcCoreTasks/calibimg.rst

```diff
@@ -0,0 +1,216 @@
+#############
+CalibrateTask
+#############
+
+Given a properly characterized exposure (which means one with a PSF
+determined and shipped along with the image, commonly done previous to
+running this Task by :doc:`CharacterizeImage <charimg>`), detect
```

Comment:
Again, I think this has too much redundancy in the words.  Isn't it sufficient to say "Given a background subtracted image, PSF, and initial astrometric solution, ..."?  You might add a parenthetical link to `CharacterizeImageTask` too.